### PR TITLE
Improve Invoice Layout

### PR DESCRIPTION
**Pull request by luxflux**
This makes the invoice layout more readable:
* Fixes ui bug when adding new line items
* Gives more space to the account selectors

screenshot 2016-01-25 17 19 04

### CODE REVIEW

File: app/views/line_items/_list_form.html.haml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 %h3= t('title.line_items')
-%table#line_items.table.table-striped{'data-direct-account-id' => resource.direct_account.try(:id), 'data-direct-account-factor' => resource.direct_account_factor}
-  %thead
-    = render 'line_items/list_header', :invoice => invoice
-  %tbody.sortable
+%div#line_items{'data-direct-account-id' => resource.direct_account.try(:id), 'data-direct-account-factor' => resource.direct_account_factor}
```
Comment (huerlisi) [inline app/views/line_items/_list_form.html.haml]: %div not needed

File: app/views/line_items/_list_form.html.haml [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
+  = render 'line_items/list_header', :invoice => invoice
+  %div.list.sortable
```
Comment (huerlisi) [inline app/views/line_items/_list_form.html.haml]: %div not needed

**Comment by huerlisi** [comment]
Very nice!

Will give it a test and probably merge;-)

**Comment by huerlisi** [comment]
It's perfect on bigger screens. But there is quite some overlap of the input fields on smaller screens. (Where 'small' is < 1200...).

Quick investigation shows that reducing horizontal padding for such media sizes on all input fields would fix it.

Otherwise a big :+1:

**Comment by luxflux** [comment]
I tried to fix it for smaller screens. But it looks strange anyway for me, the date always gets cut off.
What do you think?

**Comment by huerlisi** [comment]
It's much better on small screens, but you might wrap the paddings in a media query so on bigger screens we get normal padding.

The select boxes still have some issues. It okayish for most sizes when reloading, blut failes to resize if the windows size is changed afterwards.

I think what you implemented (+ maybe a media query for the padding) is much better than before. Proper responsiveness has probably wait 'till we come around porting to bootstrap 4...

Feel free to merge.

**Comment by huerlisi** [comment]
The date field might need a 'min-width: 5em' or similar...

**Comment by luxflux** [comment]
I tried this already and it will span over the description of a line item. I more like the version where you just see the day and month of the date and the full description. So I will merge it as it is.

Thank you!

**Pull request merged**